packages/services/src/utxos/utxos.utils.spec.ts

```diff
@@ -317,3 +320,72 @@ describe('selectUniqueUtxoIds', () => {
     ]);
   });
 });
+
+describe('getOutboundUtxos', () => {
```

Comment:
```suggestion
describe(getOutboundUtxos.name, () => {
```

minimises changes of divergence in case method ever gets renamed

Reply:
https://github.com/leather-io/mono/pull/890#discussion_r1963287123
![image](https://github.com/user-attachments/assets/9378fa69-b182-4945-9983-545b2a51c19e)